src/wmtk/attribute/TupleAccessor.hpp

```diff
@@ -30,10 +39,10 @@ class TupleAccessor : protected CachingAccessor<T>
 
     using CachingBaseType::CachingBaseType;
 
-    TupleAccessor(TupleAccessor&&) = default;
-    TupleAccessor& operator=(TupleAccessor&&) = default;
-    TupleAccessor(const TupleAccessor&) = delete;
-    TupleAccessor& operator=(const TupleAccessor&) = delete;
+    //TupleAccessor(TupleAccessor&&) = default;
+    //TupleAccessor& operator=(TupleAccessor&&) = default;
+    //TupleAccessor(const TupleAccessor&) = delete;
+    //TupleAccessor& operator=(const TupleAccessor&) = delete;
```

Comment:
delete?

Reply:
yep. forwarding of constructors wasn't working properly and i was testing.